modules/custom/openy_gxp/openy_gxp.drush.inc

```diff
@@ -20,9 +20,6 @@ function openy_gxp_drush_command() {
  * Command 'openy-gxp-sync'.
  */
 function drush_openy_gxp_sync() {
-  // First rollback all the data completely.
-  drush_migrate_tools_migrate_rollback('gxp_offerings_import');
```

Comment:
Why is this removed from drush?

Reply:
we run it as a part of form submission now. No need to rollback same migration twice